jdaviz/core/user_api.py

```diff
@@ -34,6 +35,9 @@ def __getattr__(self, attr):
         if attr in _internal_attrs or attr not in self._expose:
             return super().__getattribute__(attr)
 
+        if attr in self._deprecated:
+            logging.warning("DeprecationWarning: %s is deprecated" % attr)
```

Comment:
![Screenshot 2024-11-18 114104](https://github.com/user-attachments/assets/41307cd6-a613-407e-8475-846e1b73cc88)

Reply:
I thought about making it fancier so it can say "use meta instead" but decided against it because hopefully we won't do this often and I don't want to overcomplicate the machinery.